### PR TITLE
Added CSSmatic

### LINKED ISSUE #159: CSSmatic
[CSSmatic](http://www.cssmatic.com/)

> The ultimate CSS tools for web designers

A web-app to visualize CSS changes in browser itself using user friendly UI and lets the user download appropriate CSS or compiled SASS as per the preferences.

Currently supports 4 tools : 
* Gradient Generator
* Border Radius
* Noise Texture
* Box Shadow

The bunch of tools provided are amazing and time-saving.

This app should be appropriate for *Programming Tools* section.

### PR DESCRIPTION
**Pull request by anshumanv**
**http://www.cssmatic.com/**
Fixes #159 
**Ultimate CSS tools for web developers.**

# By submitting this pull request I confirm I've read and complied with the below requirements.

Failure to properly do so will just result in the pull request being closed and everyone's time wasted. Please read it twice. Most people miss many things.

- [x] I have read the [Contribution guidelines](https://github.com/aviaryan/awesome-no-login-web-apps/blob/master/CONTRIBUTING.md) and I am confident that my PR is suitable. 
- [x] This pull request has a descriptive title. For example, `Add [App name]`, not `Update readme.md` or `Added new entries`.
- [x] The app I added is not a duplicate.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -240,6 +240,7 @@ Powershell, Redis and lots others. Also has a good collection of IDEs for progra
 * [FAARY](http://faary.com/) - Free, Online CSS form generator. Requires form elements type and name to preview form and generate downloadable good looking form code.
 * [pForm](http://www.phpform.org/) - Create HTML forms through a simple GUI and preview or download the corresponding code.
 * [CSS Autoprefixer](https://autoprefixer.github.io/) - A postprocessor for handling vendor prefixes in CSS. Just paste the regular CSS and get the vendor prefix handled CSS instantly.
+* [CSS Matic](https://www.cssmatic.com/) - Provides predefined tools to manipulate view and gives the corresponding CSS or SASS.
```
Comment (aviaryan) [inline README.md]: *manipulate view* - It's difficult to understand what this app does from the description.

**Comment by Anshuman-Verma** [comment]
@aviaryan please review again, I think `css tools` is the important keyword we needed.

**Comment by anshumanv** [comment]
Still changes needed?

**Comment by Monal5031** [comment]
ping @aviaryan this one needs review :D

**Comment by aviaryan** [comment]
@Monal5031 Thanks for the heads up.
@anshumanv I just had a look at the site again. It supports only 4 different types of CSS tools so I think the app description should be updated to add more specificty to the information. Like what types of tools does this site have, etc..

**Comment by anshumanv** [comment]
Thanks for suggesting the changes, updated. @aviaryan

**Pull request merged**